met-api/sample.env

```diff
@@ -1,52 +1,45 @@
-JWT_OIDC_JWKS_URI=https://localhost:8080/auth/realms/met/protocol/openid-connect/certs
-JWT_OIDC_CACHING_ENABLED=True
-JWT_OIDC_JWKS_CACHE_TIMEOUT=3000000
-
-SITE_URL=http://localhost:3000
-KEYCLOAK_BASE_URL=https://localhost:8080
-KEYCLOAK_URL_REALM=met
-
-MET_ADMIN_CLIENT_ID=met-admin
-MET_ADMIN_CLIENT_SECRET=<GENERATED_CLIENT_SECRET>
-
-NOTIFICATIONS_EMAIL_ENDPOINT=https://localhost:5002/api/v1/notifications/email
-VERIFICATION_EMAIL_TEMPLATE_ID=c4cc1633-321a-4400-8a22-272acecd836a
-SUBSCRIBE_EMAIL_TEMPLATE_ID=9cd4942b-8ac9-49ae-a869-c800c57a7472
-REJECTED_EMAIL_TEMPLATE_ID=fede7ed2-4e4f-4278-9881-dadc21df8f11
-ACCESS_REQUEST_EMAIL_ADDRESS=tiago.x.graf@gov.bc.ca
-ACCESS_REQUEST_EMAIL_TEMPLATE_ID=41afa792-4c75-425a-9ad9-c558561d6669
-
-EAO_SUBMISSION_REPORT=614d387c-1fa9-4a04-af42-13a027a837d6
-EAO_ENGAGEMENT_CLOSEOUT=8c76a4dc-eec2-439b-8234-4c95234f128d
-EAO_VERIFICATION_EMAIL_TEMPLATE_ID=c08693e0-50eb-4815-9d1f-1036cdba9876
-
-
-
-CORS_ORIGIN=http://192.168.0.145:8000,http://192.168.0.145:3000,http://localhost:8000,http://localhost:3000,http://localhost:5000
-
-S3_BUCKET=<BUCKET_NAME>
-S3_ACCESS_KEY_ID=met-admin
-S3_SECRET_ACCESS_KEY=<GENERATED_S3_SECRET>
-S3_HOST=citz-gdx.objectstore.gov.bc.ca
-S3_REGION=us-east-1
-S3_SERVICE=execute-api
\ No newline at end of file
+# Database configuration. 
+# See DEVELOPMENT.md for instructions on how to set up the local database.
+DATABASE_HOST="localhost"
+DATABASE_PORT="5432"
+DATABASE_USERNAME="postgres"
+DATABASE_PASSWORD="postgres"
+DATABASE_NAME="met"
+
+# Email API endpoint
+NOTIFICATIONS_EMAIL_ENDPOINT=https://met-notify-api-dev.apps.gold.devops.gov.bc.ca/api/v1/notifications/email
+
+# Keycloak configuration. Keycloak is now hosted, and local keycloak instances are no longer needed.
+KEYCLOAK_BASE_URL=https://dev.loginproxy.gov.bc.ca/auth
+KEYCLOAK_REALMNAME=standard
+JWT_OIDC_AUDIENCE=modern-engagement-tools-4787
+JWT_OIDC_WELL_KNOWN_CONFIG=${KEYCLOAK_BASE_URL}/realms/${KEYCLOAK_REALMNAME}/.well-known/openid-configuration
+JWT_OIDC_JWKS_URI=${KEYCLOAK_BASE_URL}/realms/${KEYCLOAK_REALMNAME}/protocol/openid-connect/certs
+JWT_OIDC_ISSUER=${KEYCLOAK_BASE_URL}/realms/${KEYCLOAK_REALMNAME}
+
+# Authenticates the MET API with Keycloak for running tests.
+# Currently unused since the hosted Keycloak instance does not support API usage.
+MET_ADMIN_CLIENT_ID=
+MET_ADMIN_CLIENT_SECRET=
+
+# S3 configuration. Used for uploading custom header images, etc.
+S3_ACCESS_KEY_ID=
+S3_BUCKET=
+S3_HOST='citz-gdx.objectstore.gov.bc.ca'
+S3_REGION='us-east-1'
+S3_SECRET_ACCESS_KEY=
+S3_SERVICE='execute-api'
+
+# EPIC integration configuration
+EPIC_URL=https://eagle-dev.apps.silver.devops.gov.bc.ca/api/commentperiod
+EPIC_JWT_OIDC_ISSUER=${KEYCLOAK_BASE_URL}/auth/realms/eao-epic
+EPIC_KC_CLIENT_ID=eagle-admin-console
+EPIC_MILESTONE=5cf00c03a266b7e1877504e9
+EPIC_KEYCLOAK_SERVICE_ACCOUNT_ID=
+EPIC_KEYCLOAK_SERVICE_ACCOUNT_SECRET=
+
+# Allowed CORS origins
+CORS_ORIGIN=http://localhost:3000,http://localhost:5000
```

Comment:
I love the organization here, thanks for fixing things up.

Are there any values in here that are sensitive and shouldn't be visible via GitHub?

Reply:
Mostly the S3 secrets, and the MET_ADMIN variables. Once we get the client id/secret from Pathfinder SSO, that will also be sensitive.
However, none of that information is present in the GH release; the relevant variables were replaced with [blank definitions](https://github.com/bcgov/met-public/blob/274051ecf8f024ed47f8c45b5aca70182278d291/met-api/sample.env#L29).